frontend/packages/pipelines-plugin/src/components/pipelines/resource-types/triggers.ts

```diff
@@ -1,23 +1,6 @@
 import { K8sResourceCommon, K8sResourceKind, Toleration } from '@console/internal/module/k8s';
 import { PipelineRunKind } from '../../../types';
 
-export type TriggerKind = K8sResourceCommon & {
```

Comment:
Yikes, this file really should exist under types. My bad! I missed it when I was doing the type refactors.

Reply:
I'll leave this up to you if you want to do this part of your PR. If not, we can just put up another PR to address it.

Reply:
will do in separate PR.